README.md

```diff
@@ -1,3 +1,46 @@
 # API-Monitor
 [![CircleCI](https://circleci.com/gh/Ashaba/API-Monitor/tree/develop.svg?style=svg)](https://circleci.com/gh/Ashaba/API-Monitor/tree/develop)
 [![Coverage Status](https://coveralls.io/repos/github/Ashaba/API-Monitor/badge.svg?branch=develop)](https://coveralls.io/github/Ashaba/API-Monitor?branch=develop)
+ 
+ An application the performs health checks of an api
+ 
+ # Dependencies
+ - `Python 3.6.2`
+ - `PostgreSQL`
+ - `JQuery-3.3.1`
+ - `Flask 0.12.2`
+ 
+ # Installation
+ - Clone the repository  
+ ```$ git clone https://github.com/Ashaba/API-Monitor.git```
+ 
+ - Copy contents of  `env.sample` into `.env`   
+ ```$ cp env.sample .env```
+ 
+ - Configure and setup database in the `.env` file `SQLALCHEMY_DATABASE_URI`  
+ 
+ - Create a virtual environment  
+ ```$ virtualenv -p python3 envname```
+ 
+ - Activate the virtual environment  
+ ```$ source/envname/bin/activate```
+ 
+ - Install requirements  
+ ```$ pip install -r requirements.txt```
+ 
+ - Add environment variables to the `PYTHONPATH`  
+ ```$ export $(cat .env)```
+ 
+ # Database Migrations
+ - Run migrations  
+ 
+ ```$ python manage.py db migrate```
```

Comment:
When someone is setting up a project, they use already pushed migrations, so they don't need to run the `migrate` command.

Reply:
Done